packages/clay-multi-step-nav/docs/multi-step-nav.mdx

```diff
@@ -25,6 +28,16 @@ Each step can have two different states: `active` or `complete` defined by props
 
```

Comment:
The line above needs to be updated to match the new `state` prop: "Each step can have two different states: `active` or `complete` defined by props as you can see below."